mlrun/artifacts/dataset.py

```diff
@@ -283,14 +283,16 @@ def update_preview_fields_from_df(
         if artifact.spec.length > preview_rows_length and not ignore_preview_limits:
             preview_df = df.head(preview_rows_length)
 
-        # reset index while dropping existing index
-        # that way it wont create another index if one already there
-        preview_df = preview_df.reset_index(drop=True)
```

Comment:
Ci fails because there is another instance of such flow (code dup) @ lines 516

Reply:
It actually needs to be implemented as well. the obsoletion is pushed to mlrun 1.7 so that is needed as well.

Reply:
I have just updated it for us. Please take a look and see if it looks reasonable to you. Thanks!